controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller.go

```diff
@@ -119,8 +120,12 @@ func (r *Reconciler) ReconcileNormal(ctx goctx.Context, cclItem *imgregv1a1.Clus
 		logger.Error(nameErr, "Unsupported ClusterContentLibraryItem, skip reconciling")
 		return nil
 	}
-	cvmi := &vmopv1a1.ClusterVirtualMachineImage{}
-	cvmi.Name = cvmiName
+	cvmi := &vmopv1a1.ClusterVirtualMachineImage{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   cvmiName,
+			Labels: cclItem.Labels,
```

Comment:
We have a function defined below `setUpCVMIFromCCLItem` to set up cvmi fields from cclItem. We should probably move the label setup there too to make everything related in one place.